packages/esm-patient-list-app/src/patient-table/patient-table.component.scss

```diff
@@ -1,3 +1,48 @@
 .searchOverrides {
-    width: 50% !important;
-}
\ No newline at end of file
+    width: 250px ;
```

Comment:
I'd prefer not having the fixed `250px` here. Could this be solved otherwise (e.g. via an SCSS variable or a percentage)? 

**Disclaimer:** I don't have access to the designs on Zeplin right now, so I can only make a guess. If this exact width is actually a requirement of the design, let's keep this as-is.

Reply:
Currently, the fixed `250px` is required to remain constant on different screen sizes. Since percentages keep adjusting the width based on the parent div. This handles the requirements well.

Reply:
I'd say it should rather be `width: 100%; max-width: 250px` to avoid having an horizontal overflow in case of smaller canvas.

Reply:
@FlorianRappl I have tested `width: 100%; max-width: 250px` and adjusted accordingly.